server/code_switch/generation/cs_generation.py

```diff
@@ -0,0 +1,6 @@
+from abc import ABC, abstractmethod
+import random
+
+class CSGeneration(ABC):
```

Comment:
the super class should have the abstract method of generate + documantation of what goes in and what goes out